x-pack/plugins/infra/public/components/asset_details/tabs/profiling/profiling.tsx

```diff
@@ -39,9 +51,29 @@ export function Profiling() {
     },
   ];
 
+  if (loading) {
+    return (
+      <div
+        css={css`
+          display: flex;
+          justify-content: center;
+        `}
+      >
+        <EuiLoadingSpinner size="m" />
+      </div>
+    );
+  }
+
   return (
     <>
-      <EuiTabbedContent tabs={tabs} initialSelectedTab={tabs[0]} />
+      {error !== null && response?.has_setup === false ? (
```

Comment:
Is this a correct condition? Right now it seems like it's going to display the empty state only when there is an error and response at the same time which is mutually exclusive. Am I missing something?